bazel/repositories.bzl

```diff
@@ -221,7 +221,10 @@ def envoy_dependencies(skip_targets = []):
     external_http_archive("bazel_toolchains")
     external_http_archive("bazel_compdb")
     external_http_archive("envoy_build_tools")
-    external_http_archive("rules_pkg")
+    external_http_archive(
```

Comment:
can we remove the patch once `rules_pkg` has been updated? im thinking we might want to comment that here

Reply:
(i see we can upstream)